src/components/CreateUserForm/CreateUserForm.tsx

```diff
@@ -0,0 +1,125 @@
+import { useMutation, useQuery } from '@apollo/client';
+import CloseIcon from '@mui/icons-material/Close';
+import { Dialog, DialogContent, IconButton } from '@mui/material';
+import { useForm } from 'react-hook-form';
+
+import { ICvsResult } from '../../graphql/cvs/ICvsResult';
+import { CVS } from '../../graphql/cvs/query';
+import { IDepartmentResult } from '../../graphql/departments/IDepartmentsResult';
+import { DEPARTMENTS } from '../../graphql/departments/query';
+import { IPositionResult } from '../../graphql/positions/IpositionResult';
+import { POSITIONS } from '../../graphql/positions/query';
+import { CREATE_USER } from '../../graphql/user/mutation';
+import { USERS } from '../../graphql/users/query';
+import Preloader from '../Preloader';
+import {
+  StyledBox,
+  StyledButton,
+  StyledDialogTitle
+} from './CreateUserForm.styles';
+import FormFields from './FormFields';
+import { IFormInput } from './IFormInput';
+
+interface Props {
+  opened: boolean;
+  close: () => void;
+  confirm: () => void;
+}
+
+const UserProfileForm: React.FC<Props> = ({ close, confirm, opened }) => {
```

Comment:
It would be better to make this component reusable (we have similar modal windows on the other pages)